internal/cluster/distribution/eks/eksworkflow/activity_update_node_group.go

```diff
@@ -239,6 +240,18 @@ func (a UpdateNodeGroupActivity) Execute(ctx context.Context, input UpdateNodeGr
 		},
 	}
 
+	if input.UseInstanceStore != nil {
+		stackParams = append(stackParams, &cloudformation.Parameter{
+			ParameterKey:   aws.String("UseInstanceStore"),
+			ParameterValue: aws.String(strconv.FormatBool(*input.UseInstanceStore)),
+		})
+	} else if !input.CurrentTemplateVersion.IsLessThan("2.2.0") {
+		stackParams = append(stackParams, &cloudformation.Parameter{
+			ParameterKey:     aws.String("UseInstanceStore"),
+			UsePreviousValue: aws.Bool(true),
+		})
+	}
+
```

Comment:
**Note**: this one should be inlined as well as the `UpdateASG` was. (Otherwise I approve.)